fix(ui,content): Shorten spacediving job descriptions to avoid overflow in the Missions panel (#6181)
endless-sky/endless-sky: data/human/transport missions.txt

	job
	repeat
	name "Spacediving at <planet>"

> 	description "Head to <destination> with <bunks> professionally equipped skydivers and drop them from the edge of space.  They've agreed to pay <payment> when they land."

Reviewer: Was the two-spaces sentence separation intended?

Author: No, that was a search/replace artifact! I just fixed them. :-)

> 	description "Head to <destination> with <bunks> skydivers and drop them from the edge of space. They look like professionals. Extreme sports jobs typically carry more risks than regular transport jobs. Payment is <payment>."

Reviewer: Maybe something like this? It suggests, without overtly stating as much, that one might not get paid in full.
```suggestion
	description "Head to <destination> with <bunks>{{qualifier}} skydivers and drop them from the edge of space. They've agreed to pay <payment> when they land."
```
Qualifiers could be "professionally equipped," "professional-looking," "amateurish", "poorly equipped," or none at all, to provide concise textual flavor.

I think it's okay to not state that there is a risk of being "fined" by these missions, too.

Author: The texts here have been discussed quite a bit in the [original PR](https://github.com/endless-sky/endless-sky/pull/5781) and your suggestion on the risk seems to match the outcome of that discussion quite well.

The qualifiers do have a bit more meaning than just the "textual flavor"; the idea is that you can estimate the risk a bit better if you read the descriptions carefully. So if we use the qualifier here, then there should be separate qualifiers for the pros and for the amateurs.

	job
	repeat
	name "Spacediving at <planet>"

> 	description "Head to <destination> with <bunks> skydivers and drop them from the edge of space. They look professional, but extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is 120000."

Reviewer 2: ```suggestion
	description "Head to <destination> with <bunks> skydivers and drop them from the edge of space. They look professional, but extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is 120000 credits."
```

	job
	repeat
	name "Spacediving at <planet>"

> 	description "Head to <destination> with <bunks> skydivers and drop them from the edge of space. They look professional, but extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is 120000."

Reviewer 2: ```suggestion
	description "Head to <destination> with <bunks> skydivers and drop them from the edge of space. They look professional, but extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is 120000 credits."
```
